com.microsoft.mrtk.uxcore/Button/RectTransformColliderFitter.cs

```diff
@@ -52,6 +52,36 @@ public BoxCollider ThisCollider
         [Tooltip("2D padding around the RectTransform.")]
         private Vector2 padding = Vector2.zero;
 
+        [SerializeField]
+        [Tooltip("Whether the collider should be affected by Rect2DMasks in parent components")]
+        private bool masked = true;
```

Comment:
Style suggestion; we generally try to name the private field and public property the same for clarity. I'd recommend naming these `applyMask` and `ApplyMask`, or `UseMask`, or something along those lines! Thanks for this, it's a great feature to have.

Reply:
Thanks, updated.